components/raftstore/src/store/config.rs

```diff
@@ -159,6 +159,10 @@ pub struct Config {
     pub hibernate_regions: bool,
     #[config(hidden)]
     pub early_apply: bool,
+    #[config(hidden)]
+    pub apply_yield_duration: ReadableDuration,
+    #[config(hidden)]
+    pub reschedule_duration: ReadableDuration,
```

Comment:
How about define it as a configuration for batch system?

Reply:
I don't think it's a good idea since we use batch system as a library rather a module.

Reply:
Library can has its own configuration. The reason to move it into batch system is to avoid defining every configuration twice.

Reply:
We could change it later, there is only one option needed for batch system now.

Reply:
No, there are `batch_size`, `pool_size` and "reschedule_duration`.

Reply:
Then should we change the config name to something like `raftstore.store.pool_size` and `raftstore.apply.pool_size`?

Reply:
Serde should be able to flatten it with prefix.